projects/core/src/cms/model/content-slot-component-data.model.ts

```diff
@@ -2,5 +2,5 @@ export interface ContentSlotComponentData {
   uid?: string;
   typeCode?: string;
   flexType?: string;
-  properties?: any;
```

Comment:
I would reconsider this change, as `object` here doesn't add much in terms of type safety, but make using it harder.

Reply:
Done